pkg/alertmanager/alertmanager.go

```diff
@@ -233,7 +239,7 @@ func New(cfg *Config, reg *prometheus.Registry) (*Alertmanager, error) {
 
 	am.wg.Add(1)
```

Comment:
```suggestion
        // Run the silences maintenance in a dedicated goroutine.
	am.wg.Add(1)
```

Reply:
Done in https://github.com/grafana/mimir/pull/4099/commits/9a6e465447c77bf9810d135eae13040cfbbf891e